spec/send_recovery_link_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'send_recover_link'
+
+describe SendRecoverLink do
+ let(:user){double :user, email: "test@test.com", password_token: "12345678"}
+ let(:mail_gun_client){double :mail_gun_client}
+ let(:sandbox_domain_name) { ENV["sandbox_domain_name"] }
+
+ it "sends a message to mailgun when it is called" do
+   params = {from: "bookmarkmanager@mail.com",
+             to: user.email,
+             subject: "reset your password",
+             text: "click here to reset your password http://yourherokuapp.com/reset_password?token=#{user.password_token}" }
```

Comment:
Metrics/LineLength: Line is too long. [125/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)